woocommerce/woocommerce-android
# [Better Tablet Products] Reviews in settings have double toolbar

Closes: #11071


### Description

I noticed that reviews in the settings have double toolbar. The reason was that 2 different framgnets reused the same layout. I extracted the common part to solve this

### Testing instructions

On a tablet and a phone
* Products -> Product -> Reviews -> Notice that toolbar is single there
* Settings -> Reviews -> Notice that the toolbar is single there (it was double)

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/4d020c91-7a5a-43b5-a842-4865ac9d65b8
